src/sass/mixins/_Directionality.Mixins.scss

```diff
@@ -32,292 +32,292 @@
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-right-width: $width;
   }
 }
 
-@mixin border-radius($topLeft, $topRight, $bottomRight, $bottomLeft) {
+@mixin ms-borderRadius($topLeft, $topRight, $bottomRight, $bottomLeft) {
   border-radius: $topLeft $topRight $bottomRight $bottomLeft;
 
-  @include RTL {
+  @include ms-RTL {
     border-radius: $topRight $topLeft $bottomLeft $bottomRight;
   }
 }
 
-@mixin border-right($width, $style, $color) {
-  @include LTR {
+@mixin ms-borderRight($width, $style, $color) {
+  @include ms-LTR {
     border-right: $width $style $color;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-left: $width $style $color;
   }
 }
 
-@mixin border-right-color($color) {
-  @include LTR {
+@mixin ms-borderRightColor($color) {
+  @include ms-LTR {
       border-right-color: $color;
   }
 
-  @include RTL {
+  @include ms-RTL {
       border-left-color: $color;
   }
 }
 
-@mixin border-right-style($style) {
-  @include LTR {
+@mixin ms-borderRightStyle($style) {
+  @include ms-LTR {
     border-right-style: $style;
   }
   
-  @include RTL {
+  @include ms-RTL {
     border-left-style: $style;
   }
 }
 
-@mixin border-right-width($width) {
-  @include LTR {
+@mixin ms-borderRightWidth($width) {
+  @include ms-LTR {
     border-right-width: $width;
   }
 
-  @include RTL {
+  @include ms-RTL {
     border-left-width: $width;
   }
 }
 
 // Floats.
-@mixin clear($side) {
+@mixin ms-clear($side) {
   @if $side == left {
-    @include LTR {
+    @include ms-LTR {
       clear: $side;
     }
 
-    @include RTL {
+    @include ms-RTL {
       clear: right;
     }
   } @else if $side == right {
-    @include LTR {
+    @include ms-LTR {
       clear: $side;
     }
 
-    @include RTL {
+    @include ms-RTL {
       clear: left;
     }
   } @else {
       clear: $side;
   }
 }
 
-@mixin float($direction) {
+@mixin ms-float($direction) {
   @if $direction == left {
-    @include LTR {
+    @include ms-LTR {
       float: left;
     }
 
-    @include RTL {
+    @include ms-RTL {
       float: right;
     }
   } @else {
-    @include LTR {
+    @include ms-LTR {
       float: right;
     }
 
-    @include RTL {
+    @include ms-RTL {
       float: left;
     }
   }
 }
 
 // Positioning.
-@mixin left($distance) {
-  @include LTR {
+@mixin ms-left($distance) {
+  @include ms-LTR {
     left: $distance;
   }
 
-  @include RTL {
+  @include ms-RTL {
     right: $distance;
   }
 }
 
-@mixin right($distance) {
-  @include LTR {
+@mixin ms-right($distance) {
+  @include ms-LTR {
     right: $distance;
   }
 
-  @include RTL {
+  @include ms-RTL {
     left: $distance;
   }
 }
 
 // Margins.
-@mixin margin($top, $right, $bottom, $left) {
+@mixin ms-margin($top, $right, $bottom, $left) {
   margin: $top $right $bottom $left;
 
-  @include RTL {
+  @include ms-RTL {
     margin: $top $left $bottom $right;
   }
 }
 
-@mixin margin-left($distance) {
-  @include LTR {
+@mixin ms-marginLeft($distance) {
```

Comment:
It's certainly a 'nice to have' that all of the mixins follow the same naming convention and not an absolute must. My thinking was that since we're breaking everything by adding the "ms-" prefix anyway, it's a good opportunity to standardize the name.